src/lib/openapi-to-json-schema.ts

```diff
@@ -674,6 +683,13 @@ export default function toJSONSchema(
             refLogger,
             transformer,
           });
+
+          // If we have a non-array `required` entry in our `items` schema then it's invalid and we
+          // should remove it. We only support non-array boolean `required` properties inside object
+          // properties.
+          if (isObject(schema.items) && 'required' in schema.items && !Array.isArray(schema.items.required)) {
```

Comment:
if I'm understanding this part correctly, I don't think `required` on `items` is something our validator accepts: [bin](https://bin.readme.com/s/64b8283621693d002128f405)

Reply:
Ah cool. I'll leave this in here just in case we have some weird cases deep in the DB for it.